vassal-app/src/main/resources/VASSAL/i18n/Editor.properties

```diff
@@ -260,6 +260,7 @@ Editor.BeanShell.sum2=Sum the values of the named property in matching pieces on
 Editor.BeanShell.sum3=Sum the values of the named property in matching pieces on the named map
 Editor.BeanShell.sum4=Count the number of matching pieces on all maps
 Editor.BeanShell.sum5=Count the number of matching pieces on the named map
+Editor.BeanShell.sum6=Count the number of pieces with any non-blank value of named property, in the same stack. Or if blank property passed, returns the total number of pieces in the stack.
```

Comment:
Extra description for the no-args version. Remove the 'Blank property' blah for the 1-arg version. The code won't change, but we will direct designers to use the no-arg version if they just want a straight stack count.